impl/authentication/google.go

```diff
@@ -99,13 +104,16 @@ func setDifference(a, b []string) []string {
 	return diff
 }
 
+func parseToken(accessToken string) *oauth2.Token {
```

Comment:
Does this add much value?

Reply:
Not really, except that it nicely separates the ideas of parsing and authenticating.